draft-ietf-tls-esni.md

```diff
@@ -994,6 +966,72 @@ The "ech_is_inner" is expected to have an empty payload. If the payload is
 non-empty (i.e., the length of the "extension_data" field is non-zero) then the
 backend server MUST abort the handshake with an "illegal_parameter" alert.
 
+## Handling HelloRetryRequest {#server-hrr}
```

Comment:
"Handling" is wrong here. You are generating.

Reply:
Changed to "Sending".